packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```diff
@@ -142,16 +148,23 @@ sh %{_sourcedir}/NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}.run -x
 # Move to the sources directory and apply patch
 pushd NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}
 %patch 1 -p1
-cp -r kernel-open grid
 popd
 
+# Extract GRID drivers just like Tesla
+%if "%{_cross_arch}" == "x86_64"
+sh %{_sourcedir}/NVIDIA-Linux-x86_64-%{tesla_ver}-grid-aws.run -x
```

Comment:
Not worth making it inconsistent with the other drivers, but if we ever refactor these specs, this should be equivalent:
```
sh %{S:2} -x
```

Reply:
Makes sense. The fact this is only x86_64 makes it a bit different either way but this make the file feel consistent at least.